src/popup/Popup.tsx

```diff
@@ -23,23 +26,19 @@ export default function Popup(): JSX.Element {
     });
   }, [emotesEnabled]);
 
-  const emotesButton = emotesEnabled ? (
-    <input
-      type="checkbox"
-      name="emotesCheckbox"
-      onChange={changeEmoteStatus}
-      checked
-    />
-  ) : (
-    <input type="checkbox" name="emotesCheckbox" onChange={changeEmoteStatus} />
-  );
-
   return (
-    <div style={{ width: '150px' }}>
-      <h3>tt.fm+</h3>
-      <br />
-      Twitch Emotes
-      {emotesButton}
-    </div>
+    <Container w="25em" h="15em" bgColor="#243454" color="#a37b04">
+      <Heading>tt.fm+</Heading>
+      <Divider mb="1em" />
+      <VStack justifyContent="space-between">
+        <LabeledSwitch
```

Comment:
Great replacement of the ternary `<input />` code, much cleaner. However, when you click on the icon to open the popup it causes a warning to appear. (see `chrome://extensions`)